### PR TITLE
Update Primitives2D.py

### PR DESCRIPTION
**Pull request by PipKat**
Not sure if something in 2D can  have x, y, and z coordinates so check the parameters "position" and "dimension_list"

### CODE REVIEW

File: pyaedt/modeler/Primitives2D.py [hunk 1]
```diff
@@ -28,21 +44,22 @@ def create_circle(self, position, radius, numSides=0, name=None, matname=None):
 
         Parameters
         ----------
-        position :
-            ApplicationName.modeler.Position(x,y,z) object
+        position : list
+            Center point of the circle in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives2D.py:48]:
```suggestion
            Center point of the circle in a list of ``[x, y, z]`` coordinates.
```

File: pyaedt/modeler/Primitives2D.py [hunk 2]
```diff
@@ -74,24 +91,27 @@ def create_ellipse(self,position, major_raidus, ratio, bIsCovered=True, name=Non
 
         Parameters
         ----------
-        position :
-            ApplicationName.modeler.Position(x,y,z) object
+        position : list
+            Center point of the ellipse in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives2D.py:95]:
```suggestion
            Center point of the ellipse in a list of ``[x, y, z]`` coordinates.
```

File: pyaedt/modeler/Primitives2D.py [hunk 3]
```diff
@@ -125,19 +145,20 @@ def create_rectangle(self, position, dimension_list, name=None, matname=None):
 
         Parameters
         ----------
-        position :
-            ApplicationName.modeler.Position(x,y,z) object
+        position : list
+            Center point of the rectangle in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives2D.py:149]:
```suggestion
            Center point of the rectangle in a list of ``[x, y, z]`` coordinates.
```

File: pyaedt/modeler/Primitives2D.py [hunk 4]
```diff
@@ -125,19 +145,20 @@ def create_rectangle(self, position, dimension_list, name=None, matname=None):
...
         dimension_list : list
-            dimension list
-        name :
-            Name of the object. The default is ``None``.
+            Dimensions of the rectangle in a list of [x, y, z] coordinates.
```
Comment (PipKat) [inline pyaedt/modeler/Primitives2D.py:151]:
```suggestion
            Dimensions of the rectangle in a list of ``[x, y, z]`` coordinates.
```

File: pyaedt/modeler/Primitives2D.py [hunk 5]
```diff
@@ -3,11 +3,27 @@
...
+    """
 
     @aedt_exception_handler
     def is3d(self):
-        """Returns False always to indicate a 3D analysis type."""
+        """Check if the analysis is a 3D type.
+        
+        Returns
+        -------
+         ``True`` when successful, ``False`` when failed.
```
Comment (akaszynski) [inline pyaedt/modeler/Primitives2D.py:24]:
```suggestion
        bool
            ``True`` when successful, ``False`` when failed.
```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

**Pull request merged**